andersoncd/estimators.py

```diff
@@ -5,10 +5,114 @@
 from sklearn.linear_model import Lasso as Lasso_sklearn
 from sklearn.linear_model import ElasticNet as ElasticNet_sklearn
 
-from andersoncd.penalties import WeightedL1, L1_plus_L2, MCP_pen
+from andersoncd.penalties import L1, WeightedL1, L1_plus_L2, MCP_pen
 from andersoncd.solver import solver_path
 
 
+class Lasso(Lasso_sklearn):
+    r"""
+    Lasso estimator based on Celer solver and primal extrapolation.
+
+    The optimization objective for Lasso is::
+
+    (1 / (2 * n_samples)) * ||y - X w||^2_2 + alpha * \sum_j |w_j|
+
+    Parameters
+    ----------
+    alpha : float, optional
+        Constant that multiplies the L1 term. Defaults to 1.0.
+        ``alpha = 0`` is equivalent to an ordinary least square.
```

Comment:
I would remove these 3 lines, I find it too verbose (I removed it in the other classes)